libtransmission/torrent.h

```diff
@@ -712,6 +715,8 @@ struct tr_torrent : public tr_completion::torrent_view
 
     tr_labels_t labels;
 
+    std::string group;
```

Comment:
possibly use tr_interned_string here so that the string is allocated once, rather than once per torrent in the group?

Reply:
Interned in #2783